src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/Button/ButtonHandlerTests.cs

```diff
@@ -7,7 +7,7 @@ namespace Xamarin.Platform.Handlers.DeviceTests
 {
 	public partial class ButtonHandlerTests : HandlerTestBase<ButtonHandler>
 	{
-		[Fact]
+		[Fact(DisplayName = "Text Initializes Correctly")]
```

Comment:
Can we maybe add some sort of "type" here so that we know what component has failed? Is this the button or the label or some other control. We are going to have a good few controls, and all the messages will look the same.

Reply:
Good point, updated!